include/rwqueue.h

```diff
@@ -134,37 +132,52 @@ class RWQueue {
 	// (the operation will be done in chunks, provided pressure on the other
 	// side is relieved).
 
-	// Items are std::move'd out of the source vector into the queue. This
-	// function clear()s the vector such that it's in a defined state on
-	// return (and can be reused). The method potentially blocks until there
-	// is enough capacity in the queue for the new items.
+	// Items are std::move'd out of the source vector and also erased so the
+	// source vector is returned in a defined state. The method potentially
+	// blocks until there is enough capacity in the queue for the source items.
 
 	// If queuing has stopped prior to bulk enqueing, then this will
-	// immediately return false and no items will be queued.
+	// immediately return with 0 and no items will be queued.
```

Comment:
Prefer simple language: "If the queue is stopped, ..."

Reply:
fixed:

```c++
// If the queue is stopped then the function unblocks and returns the
// quantity enqueued (which can be less than the number requested).
```